apps/web/src/components/Container.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactNode } from 'react';
+
+//  px-4 sm:px-6 md:px-8 lg:px-12 xl:px-16
+export interface ContainerProps {
+  children: ReactNode;
+}
+
+const Container = ({ children }: ContainerProps) => {
+  return <div className="flex flex-col w-full max-w-7xl mx-auto">{children}</div>;
+};
```

Comment:
Let's set a custom css variable to control the container width called `--container-max-width`

Those widths should be:
- Portrait (default): 22.375rem /* 358px */
- Landscape: 82.0625rem /* 1312px */

```suggestion
const Container = ({ children }: ContainerProps) => {
  return <div className="flex flex-col w-full max-w-(--container-max-width) mx-auto">{children}</div>;
};
```